src/pages/products/components/ProductList.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+const ProductList = ({ data }) => {
+  return (
+    <li className="product" key={data.id}>
+      <div className="productImageBox">
```

Comment:
해당 부분은 구조분해 할당해서 사용할 수 있겠습니다.

```suggestion
const ProductList = ({ data }) => {
  const { id, name, price} = data;
  return (
    <li className="product" key={id}>
      <div className="productImageBox">
```

이런 식으로 구조분해 할당 한 뒤 값을 사용하게 되면
가독성과 유지보수 측면에서 좋습니다!